### PR TITLE
sysrepo: update to 1.4.2

### PR DESCRIPTION
**Pull request by lucize**
Signed-off-by: Lucian Cristian <lucian.cristian@gmail.com>

Maintainer: @mislavn
Tested aarch64

### CODE REVIEW

**Comment by neheb** [comment]
```
cp: cannot stat '/home/build/build_dir/build_dir/target-mips_24kc_musl/sysrepo-1.4.2/src/libsysrepo.so*': No such file or directory
```

**Comment by neheb** [comment]
sounds like you should use PKG_INSTALL.

**Comment by lucize** [comment]
there is more about this, I just need to see how to test with the new changes and I'll update.

File: libs/libyang/Makefile [hunk 1]
```diff
@@ -34,6 +34,14 @@ define Package/libyang
   DEPENDS:=+libpcre +libpthread
 endef
 
+define Package/libyang-cpp
```
Comment (neheb) [inline libs/libyang/Makefile:37]: Needs a PKG_RELEASE bump.

**Comment by lucize** [comment]
@neheb done

**Comment by neheb** [comment]
Netopeer2 is still broken. Can you look into it?

https://downloads.openwrt.org/snapshots/faillogs/powerpc_8540/packages/Netopeer2/compile.txt

**Comment by lucize** [comment]
@neheb do I need some deprecated api patch here ?

 CMakeFiles/netopeer2-cli.dir/commands.c.o: in function `parse_cert':
commands.c:(.text+0x1b44): undefined reference to `X509_get_notAfter'

https://github.com/CESNET/Netopeer2/releases 1.1.27

**Comment by lucize** [comment]
nevermind

**Comment by neheb** [comment]
I think I submitted that patch upstream..

**Pull request merged**